test/create.test.ts

```diff
@@ -208,28 +197,23 @@ describe("commands/create", function () {
     ])
       .command(mod)
       .parse();
-    assert.calledWith(
-      consoleLog,
-      match(function (value: any) {
-        value = JSON.parse(value);
-        const { prefix, name, chainId, tableId, transactionHash } =
-          value.meta.txn;
-        return (
-          prefix === "first_table" &&
-          chainId === 31337 &&
-          name.startsWith(prefix) &&
-          name.endsWith(tableId) &&
-          typeof transactionHash === "string" &&
-          transactionHash.startsWith("0x")
-        );
```

Comment:
Using the `calledWith` sinon assertion was making it hard to determine why a test was failing. I replaced this kind of single assertion with multiple checks, with multiple equal assertions from the node.js core package.
This means when one of the assertions fails the test runner will let you know which assertion it was without needing to add logging or edit the tests.